src/org/opendatakit/briefcase/export/Model.java

```diff
@@ -35,7 +35,6 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
-import java.util.function.Consumer;
```

Comment:
new/revisit

Now that we're using stuff from export in pull&push, we need to come back and revisit the whole parsing+XmlElement+Model design.